fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -1140,17 +1141,26 @@ private long getScanRangeLength(final TScanRange scanRange) {
 
     private void computeColocateJoinInstanceParam(PlanFragmentId fragmentId, int parallelExecInstanceNum, FragmentExecParams params) {
         Map<Integer, TNetworkAddress> bucketSeqToAddress = fragmentIdToSeqToAddressMap.get(fragmentId);
```

Comment:
Maybe `bucketSeqToScanRang` should be a map like `fragmentIdToSeqToAddressMap` to solve this problem is a
better understanding  way？

Reply:
I will leave it when refactoring them.